aikau/pom.xml

```diff
@@ -55,6 +55,80 @@
                </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>unitTestsGrid</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.tomcat.maven</groupId>
+                        <artifactId>tomcat7-maven-plugin</artifactId>
+                        <version>2.2</version>
+                        <executions>
+                            <execution>
+                                <id>run-embedded-tomcat</id>
+                                <!-- Bind to the pre-integration-test phase -->
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <!-- Run will run the project as a dynamic webapp -->
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <fork>true</fork>
+                                    <port>8089</port>
+                                    <ignorePackaging>true</ignorePackaging>
+                                    <warSourceDirectory>${basedir}/target/test-classes/testApp</warSourceDirectory>
+                                    <!-- The context file allows us to map static resources in a virtual webapp -->
+                                    <contextFile>${project.basedir}/tomcat/context.xml</contextFile>
+                                    <useTestClasspath>true</useTestClasspath>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>shutdown-embedded-tomcat</id>
+                                <!-- Bind to the post-integration-test phase -->
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <!-- Safely stop tomcat when integration-test phase is done -->
+                                    <goal>shutdown</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                      <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
grunt call included in the profile so it doesn't affect the integration-test phase outside the unit tests on grid context.